src/metabase/public_settings.clj

```diff
@@ -113,6 +113,20 @@
   :visibility :public
   :audit      :getter)
 
+
+  (defsetting help-link
+  (deferred-tru "TODO")
+  :default    ""
+  :type       :string
```

Comment:
From looking at other settings that is also an option, they use `:keyword` e.g.
https://github.com/metabase/metabase/blob/08480aac9416ea03ba5ae525f67b27da2ea54112/src/metabase/public_settings.clj#L397-L403

```suggestion
  :type       :keyword
```

Reply:
I don't see this being addressed yet FYI.

Reply:
Ah yeah, that's for the new version that only have 1 settings. Gotcha.